Dockerfile-windows-nano

```diff
@@ -0,0 +1,10 @@
+FROM microsoft/dotnet:2.1.403-sdk-nanoserver-1803
+
+ARG CAKE_VERSION=0.30.0
```

Comment:
The linux counter part gets version from last tag in repo so version isn't known by orchestrating cake script.

Omitting version will install latest published.

Cake 0.31.0 version command will just include version, which will make it easier to get version string in an xplat way.

Reply:
I was a little unsure exactly what to do, but if you're fine with latest, I'll change it

Reply:
Well need to get version in a good way, but we could fix that when 0.31.0 is released.

Reply:
👍